pkg/vsphere/vm/vm.go

```diff
@@ -272,17 +272,26 @@ func (vm *VirtualMachine) DeleteExceptDisks(ctx context.Context) (*types.TaskInf
 
 	op := trace.FromContext(ctx, "DeleteExceptDisks")
 
+	op.Debugf("Getting list of the devices for VM %s", vm)
```

Comment:
The VM name could contain spaces. Maybe `%q` would help readability in cases like that?

Reply:
Wouldn't have thought that `%q` would be helpful in cases like this. Great suggestion, noted!

Reply:
yes, but that is not really necessary. if you are worried about automatic log parsing, our logs needs to be redesigned anyway.

Reply:
I'm worried about human readers. If I have a VM named something as innocuous as "VCH-0-1234 (1)", the log message is:
> Getting list of the devices for VM VCH-0-1234 (1)

That could be easily mis-interpreted to mean that the VM is named "VCH-0-1234" and the list of device has one element in it, or that this is step 1 of some process. Even if they don't misinterpret it, a reader might find it ambiguous. They could reference the source code to interpret the message, but that would be a sign that our log message isn't as useful as it could be.